From a36b769cf17e87d2c59a28429e3d1ef0706a42bf Mon Sep 17 00:00:00 2001
From: Larry Gritz <lg@larrygritz.com>
Date: Wed, 14 Mar 2018 09:48:41 -0700
Subject: [PATCH] MSVC: use /bigobj on some other large files (#1900)

---
 src/libOpenImageIO/CMakeLists.txt | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/src/libOpenImageIO/CMakeLists.txt b/src/libOpenImageIO/CMakeLists.txt
index 5ffc48d..8de39c7 100644
--- a/src/libOpenImageIO/CMakeLists.txt
+++ b/src/libOpenImageIO/CMakeLists.txt
@@ -151,6 +151,9 @@ endif ()
 
 if (MSVC)
     set_property (SOURCE imagebufalgo_pixelmath.cpp APPEND_STRING PROPERTY COMPILE_FLAGS " /bigobj ")
+    set_property (SOURCE imagebufalgo_addsub.cpp APPEND_STRING PROPERTY COMPILE_FLAGS " /bigobj ")
+    set_property (SOURCE imagebufalgo_muldiv.cpp APPEND_STRING PROPERTY COMPILE_FLAGS " /bigobj ")
+    set_property (SOURCE imagebuf.cpp APPEND_STRING PROPERTY COMPILE_FLAGS " /bigobj ")
 endif ()
 
 if (WIN32)
-- 
2.5.3

